prov/usnic/src/fi_ext_usnic.h

```diff
@@ -42,7 +42,7 @@
 #include <stdint.h>
 #include <net/if.h>
 
-#define FI_PROTO_RUDP 100
+#define FI_PROTO_RUDP (100 | FI_PROV_SPECIFIC)
```

Comment:
Sorry for the after-the-merge review, but does this need to be `(100u | FI_PROV_SPECIFIC)` instead?  Or better, yet, since `FI_PROV_SPECIFIC` is a higher-order bit: `(FI_PROV_SPECIFIC | 100u)`...